### ISSUE TITLE
LLVM.GetTargetFromTriple in OO api

### ISSUE THREAD

**Issue by lukechu10** [microsoft/LLVMSharp#142]
There does not seem to be an equivalent method in the object oriented api.

**Comment by jangofett4** [comment]
Yes I also realized this, this is the only way I could make it work:
```cs
unsafe
{
    fixed (byte* ptr = Encoding.ASCII.GetBytes("data-layout"))
        LLVM.SetDataLayout(Module, (sbyte*)ptr);
    fixed (byte* ptr = Encoding.ASCII.GetBytes("target"))
        LLVM.SetTarget(Module, (sbyte*)ptr);
}
```
Also `GetBitcodeModule` functions seem to be not ported to object orinted API either.

**Comment by lukechu10** [comment]
> Yes I also realized this, this is the only way I could make it work:
> 
> ```cs
> unsafe
> {
>     fixed (byte* ptr = Encoding.ASCII.GetBytes("data-layout"))
>         LLVM.SetDataLayout(Module, (sbyte*)ptr);
>     fixed (byte* ptr = Encoding.ASCII.GetBytes("target"))
>         LLVM.SetTarget(Module, (sbyte*)ptr);
> }
> ```
> 
> Unsafe, but works.
> Also `GetBitcodeModule` functions seem to be not ported to object orinted API either.

Thanks a lot. I have been looking for a workaround for weeks now. 😃

**Comment by jangofett4** [comment]
Yeah, no problem 😉. I want create a new pull request for these missing methods but I need to research the code structure more.

**Comment by tannergooding** [comment]
The nicer managed wrapper tries to mirror the C++ API structure where possible, there are naturally chunks missing and so not everything can be mirrored perfectly.

**Comment by tannergooding** [comment]
This API exists today via `LLVMTargetRef.GetTargetFromTriple` so you don't need the raw pointer workaround. `LLVMSharp.Target` will eventually expose this too, but doesn't yet
